extension/goplugin/schemas.go

```diff
@@ -262,12 +262,28 @@ func (schema *Schema) StateFetchRaw(id string, requestContext goext.Context) (go
 	return tx.StateFetch(goext.GetContext(requestContext), schema, goext.Filter{"id": id})
 }
 
+// FetchFilterRaw returns a pointer to raw resource, containing db annotations
+func (schema *Schema) FetchFilterRaw(filter goext.Filter, requestContext goext.Context) (interface{}, error) {
```

Comment:
I have two comments related to this function.

1. Probably, you added new function here with consideration of backward compatibility. However, I think it's ok to break the compatibility so far.
1. Even if we decide to add new FetchFilterRaw function, we need `id` as a parameter then fill it in goext.Filter .

Reply:
There are some use cases where both versions could be useful.
If you want to find related object you could filter by id without having to write
```goext.Filter{"id": id}```.
Fetch with filter is more general and can be used instead of List with limit 1.